File: api/code-samples/python/eth/others/getRawTransactionByBlockNumberAndIndex.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

blockNumber = 118593751
index = "0x0"

sdk = OpenSDK(host)
eth_response = sdk.eth.get_raw_transaction_by_block_number_and_index(blockNumber, index)

print(json.loads(eth_response.response.data))

File: web3rpc/rpc-specs/code-samples/javascript/eth/transaction/getRawTransactionByBlockNumberAndIndex.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    const blockNumber = 118593751
    const index = '0x0'
    
    provider.eth.getRawTransactionByBlockNumberAndIndex(blockNumber, index, {}, (err, data, response) => {
        console.log(data);
    });

}
)()